examples/00-fluent/species_transport.py

```diff
@@ -244,7 +245,12 @@
 print(f"Cp (specific heat) option: {mixture_material.specific_heat.option()}")
 print(f"Thermal conductivity value: {mixture_material.thermal_conductivity.value()}")
 print(f"Viscosity value: {mixture_material.viscosity.value()}")
-print(f"Mass diffusivity value: {mixture_material.mass_diffusivity.value()}")
+if solver.get_fluent_version() < FluentVersion.v252:
+    print(f"Mass diffusivity value: {mixture_material.mass_diffusivity.value()}")
+else:
+    print(
+        f"Mass diffusivity value: {mixture_material.mass_diffusivity.constant_mass_diffusivity()}"
```

Comment:
@mkundu1 @seanpearsonuk @prmukherj 

This example will be published in Examples section on docs website as is in next PyFluent release 0.30.0 but at that time I don't think 25R2 will be released too. Even if we consider that 25R2 is released then we keep code for latest release only (25R2). Please let us know if you have already considered this case and your plan about it. I just want to make sure that whether we want to keep code based on Fluent version check in public examples scripts or not. Thank you.